nmdc_schema/migration_recursion.py

```diff
@@ -151,18 +97,6 @@ def main(schema_path, input_path, output_path, salvage_prefix, migrator_name):
 
     logger.info(curie_slots)
 
-    curie_migrator = CurieMigrator()
-    curie_migrator.forced_prefix = salvage_prefix
-
-    # iterate over collections, applying CURIe fixes
-    for tdk, tdv in total_dict.items():
-        logger.info(f"Fixing CURIes in {tdk} if necessary")
-
-        total_dict[tdk] = curie_migrator.fix_curies_recursively_by_key(
-            tdv, set(curie_slots))
-
-    # Instantiate an adapter that the migrator can use to manipulate a "database" represented as a Python dictionary.
-    # Also, specify some callback functions, so we can react when certain events occur.
```

Comment:
I find these two comments helpful and would prefer that they remain in the code.

Reply:
OK, where do you want them to live? All of the nearby code has been removed

Reply:
Based on how I interpret the diff, I don't think the code that was originally described by that comment has been removed.

![image](https://github.com/microbiomedata/nmdc-schema/assets/134325062/5aac197f-df80-4cc0-931d-7be51f012535)

Reply:
When I said "two comments," I meant the single "two-line comment" (on lines 164-165).